tags/resources/toc-supporting-guides/tech-papers.md

```diff
@@ -0,0 +1,154 @@
+
+### Defining a need
+
+For any individual or group interested in developing a technical paper, there must be a defined need or observable opportunity to close a knowledge gap. When proposing a technical paper, an initial scope must be established to facillitate discussion and interest within the TAG via an Issue. It is expected that the Issue title and content will be adjusted and modified as the TAG and group refine the scope.
+
+### Presenting the paper idea to the TAG
+
+The paper scope and topic should be raised in at least one TAG meeting to solicit volunteers (minimally 4). TAG leadership reserves the right to hold off on moving forward with a paper due to existing TAG commitments, lack of volunteers, or lack of consensus on need or disparities in scope. 
+
+### TAG leader sponsorship and cross-TAG awareness
+
+A TAG leader (co-chair or tech lead) must step forward to sponsor the paper. Other TAG leaders may serve as the lead for paper development but the role of sponsor must be independent from the lead for the paper to reduce any bias in the process. The paper's sponsor is responsible for ensuring the development of the paper adheres to this process, is still progressing towards publication, and provides the initial review of the draft before public comment period may open.
+
+The paper sponsor is also expected to coordinate awareness of the paper development and timelines to other TAGs to ensure they are appropriately included and have the opportunity to contribute and participate.
+
+### Refining the scope and audience and participants
+
+Interest in participating is expressed on the Issue by comments of those individuals. Soliciting interest and participation must allow for sufficient time (no less than two weeks) for fair opportunity to gather interest. Interested parties should then meet to define timelines, refine the scope, declare the audience, and determine paper lead. It is recommended that TAGs leverage communication channels to increase awareness of the proposed paper, its topic, and to solict potential contributors.
+
+## Managing paper development
+
+Once the group has met to define timelines, scope, audience, and determined the lead, the Issue must be updated to provide the following information (initial estimates provided but may be adjusted by the group or TAG leadership as appropriate):
+
+| Milestone | Estimated Time |
+| --- | --- |
+| Audience, Goals, & refining scope | 1 week |
+| Outline | 2 weeks |
+| Tasking Assignment | 1 week |
+| Content Rough-in | 2-3 weeks |
+| Collaborative Review | 2 weeks |
+| Executive Summary and content wrap up | 2 weeks |
+| Narrative Voice | 1-2 weeks |
+| Final Group Review | 1 week |
+| Community Review | 2 weeks |
+| Public comment adjudication | 2 weeks (simultaneous with review) |
```

Comment:
Looks like "Leadership Review" is missing just after "Public comment adjudication" - suggest 1-2 weeks for timeline